src/main/scala/fundamentals/level04/LogParser.scala

```diff
@@ -62,9 +71,19 @@ object LogParser {
     * scala> parseLog("X blblbaaaaa")
     * = UnknownLog("X blblbaaaaa")
     *
-    * Hint: use the `split` method on `String` and pattern match on `Array`
+    * Here is the beginning of one possible approach.
     **/
-  def parseLog(str: String): LogMessage = ???
+  def parseLog(str: String): LogMessage = {
+    val fields = str.split(",").toList
+    val logOpt: Option[LogMessage] = fields match {
```

Comment:
Can we rename to `optLog`? I think that's the convention we use in the codebase.